### ISSUE TITLE
USB0 and USB1 IP address change?   

### ISSUE THREAD

**Issue by stk21** [FalconChristmas/fpp#1145]
USB0 and USB1 devices on rPi and BB devices are tagged to 192.168.6.2 and 192.168.7.2.

Most home networks are all in the 192.168.x range.  There are some home networks that are a .7 network by default.

Would it be a good idea to change the USB0 and USB1 networks to be something different?

Perhaps 172.16.17.x and 172.16.16.x.  or a 10.254.17.x and 10.254.16.x.
Even a very high 192.168.x subnet would work as well.  192.168.254.x and 192.168.253.x

Basically just changing the /etc/default/bb-boot file.  And changing the manual to reflect.  

Just a thought.

**Comment by darylc** [comment]
192.168.7.x is the standard Beaglebone USB IP range.  I can't see us deviating from that.
